src/components/Comments.jsx

```diff
@@ -0,0 +1,37 @@
+import { fetchCommentsByArticleID } from "../api";
+import { useEffect } from "react";
+
+const Comments = ({ article_id, comments, setComments }) => {
+  useEffect(() => {
+    if (typeof article_id !== "undefined" && article_id !== "") {
```

Comment:
we should handle loading state and potential  error state for comments, as these are api calls they're not forced to work perfectly and the user should have some feedback on whether or not they need to refresh etc